account_invoice_start_end_dates/views/account_move.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2013-2016 Akretion, Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
```

Comment:
odoo tag is not needed if we don`t have "noupdate=1"

Reply:
removing odoo breaks the view
an xml file always needs a single root element
removing the root "odoo" in this view results in 3 root elements "record"
but evenin product.xml, with only 1 record element it seems to break with  an invalid structure error.
So I guess Odoo always expects a root "odoo" element

Reply:
Sorry, my mistake here - I have messed it up with "data"